src/containers/NamespaceDropdownContainer.tsx

```diff
@@ -20,7 +20,7 @@ const mapDispatchToProps = (dispatch: Dispatch<any>) => {
       dispatch(NamespaceActions.fetchNamespacesIfNeeded());
     },
     onSelect: (namespace: Namespace) => {
-      dispatch(NamespaceActions.setActiveNamespace(namespace));
+      dispatch(NamespaceActions.toggleActiveNamespace(namespace));
```

Comment:
Yeah, I'm not sure about this...

Reply:
Ok ok, i'm going to try a timed-based approach, one second and the event is going to fire.

Reply:
I guess I was thinking we could wait until the dropdown is closed but I'm OK if you think it works well as is, without a timer.  To be honest, I'm not sure how often users will actually add lots of namespaces in one go, so  it may not even matter.